test/test-api-getreport.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
-// Testcase for returning NodeReport as a string via API call
+// Testcase for returning report as a string via API call
 
 const common = require('./common.js');
 const tap = require('tap');
 const nodereport = require('../');
 var report_str = nodereport.getReport();
 common.validateContent(report_str, tap, {pid: process.pid,
-  commandline: [process.argv0, 'test/test-api-getreport.js'].join(' ')
-});
\ No newline at end of file
+  commandline: [process.argv[0], 'test/test-api-getreport.js'].join(' ')
```

Comment:
Couldn't you just check that `test/test-api-getreport.js` exists, and try `test\test-api-getreport.js` otherwise (and fail if neither exist)?

`process.argv[1]` seems like the way to go for the finding the project root.